web/src/style-helpers.ts

```diff
@@ -102,6 +102,13 @@ export const mixinResetButtonStyle = `
   margin: 0;
   font-family: inherit;
   cursor: pointer;
+
+  // undo Material UI's Button styling
+  letter-spacing: normal;
+  min-width: 0px;
+  text-transform: none;
+  line-height: normal;
+  font-size: ${FontSize.smallest};
```

Comment:
i think there might be some fun material ui click animations to reset too. 😬 it's pretty subtle, but i noticed this kind of radiating shadow when i clicked on the trigger mode toggle (on both views) and the restart button (on detail view only). i don't think this needs to block this pr.
![2021-08-20 14 02 45](https://user-images.githubusercontent.com/23283986/130275495-34ab4643-7f0a-430a-a182-24c2aede1d58.gif)

Reply:
I think this is taken care of by `disableRipple` in f0906f8